packages/amplify-graphql-transformer-core/API.md

```diff
@@ -377,6 +377,13 @@ export class InvalidTransformerError extends Error {
 // @public (undocumented)
 export const isAmplifyDynamoDbModelDataSourceStrategy: (strategy: ModelDataSourceStrategy) => strategy is AmplifyDynamoDbModelDataSourceStrategy;
 
+// @public (undocumented)
+export const isBuiltInGraphqlNode: (obj: DefinitionNode) => obj is ObjectTypeDefinitionNode | (InterfaceTypeDefinitionNode & {
```

Comment:
This already existed, but I'm now exporting it so we can use it in the construct utility. As with other utilities, it's an internal helper not intended for customer use.